src/elements/fields/CheckboxField.jsx

```diff
@@ -1,6 +1,156 @@
-import React from 'react';
+import React, { useMemo } from 'react';
 import Form from 'react-bootstrap/Form';
 
+// Draws a checkmark, similar in dimensions to the default Chrome checkbox, in CSS
+const checkmarkClipPath =
+  'polygon(40% 85%, 89% 23%, 76% 12%, 39% 59%, 23% 44%, 12% 55%)';
+
+const MIN_CHECKBOX_PX = 13;
+
+const scaleCheckboxSize = (fontSize) => {
+  return Math.max(Math.floor(fontSize * 0.5), MIN_CHECKBOX_PX);
+};
+
+const applyHeightAndWidthByFontSize = (applyStyles, target) => {
```

Comment:
this seems generic enough that we could maybe move this into the ApplyStyles class with the other style helper functions? e.g. radio button groups will probably need to use the same logic

Reply:
Yes. Let's move it when the time comes.